### PR TITLE
WIP: Develop linting fix

### PR DESCRIPTION
**Pull request by tbkr**
Fix linting issues in the CICD

### CODE REVIEW

**Comment by FussyDuck** [comment]
[CLA assistant check](https://bol-claassistant.ecmwf.int:443/ecmwf/pyfdb?pullRequest=19) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://bol-claassistant.ecmwf.int:443/ecmwf/pyfdb?pullRequest=19) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://bol-claassistant.ecmwf.int:443/check/ecmwf/pyfdb?pullRequest=19) it.</sub>

File: .github/workflows/check-publish.yaml [hunk 1]
```diff
@@ -85,11 +85,12 @@ jobs:
...
-            ecmwf/ecbuild@refs/tags/2021.08.0
-            ecmwf/eccodes@refs/tags/2.24.2
-            ecmwf/eckit@refs/tags/1.18.7
-            ecmwf/metkit@refs/tags/1.8.10
-            ecmwf/fdb@refs/tags/5.10.2
+            ecmwf/ecbuild@refs/tags/3.8.2
+            MathisRosenhauer/libaec@refs/tags/v1.1.2
+            ecmwf/eccodes@refs/tags/2.34.1
+            ecmwf/eckit@refs/tags/1.25.2
+            ecmwf/metkit@refs/tags/1.10.04
```
Comment (danovaro) [inline .github/workflows/check-publish.yaml]: can you please update to the current master (1.11.7)?

File: pyfdb/pyfdb.py [hunk 2]
```diff
@@ -12,14 +11,13 @@
...
+import os
 
 import cffi
-import os
-from pkg_resources import parse_version
 import findlibs
+from pkg_resources import parse_version
 
-__version__ = '0.0.1'
+__version__ = "0.0.1"
```
Comment (danovaro) [inline pyfdb/pyfdb.py:20]: the current master version is tagged 0.0.3
Can you please update here?

**Comment by tbkr** [comment]
Not needed anymore as. Resolved as part of the ci migration.

**Pull request closed without merging**